packages/core/services/google-maps-api-wrapper.ts

```diff
@@ -105,15 +105,16 @@ export class GoogleMapsAPIWrapper {
   }
 
   /**
-   * Creates a Google Map transit layer instance add it to map
-   * @param {TransitLayerOptions} options - TransitLayerOptions options
-   * @returns {Promise<TransitLayer>} a new transit layer object
+   * Creates a MapLayer instance for a map
+   * @param {MapLayerOptions} options - used for setting layer options
+   * @param {string} name - the type of map layer to create
+   * @returns {Promise<MapLayer>} a new map layer object
    */
-  createTransitLayer(options: mapTypes.TransitLayerOptions): Promise<mapTypes.TransitLayer>{
```

Comment:
I really think that the google-maps-api-wrapper class should have separate methods for createTransitLayer, createBicycleLayer, createTrafficLayer since there is nothing in common between these, and it's bad practice to separate code execution by a passed in string.

Reply:
Okay. I decided to go with one layerManager because Bicycling Layer and and Transit Layer has the exact methods. There is nothing different apart from the class names. I thought it would be an overkill to have 3 services with the same methods and properties, but I will separate them since that's how google does it.

Reply:
I agree to have one uniting manager. but I think there should be different methods in ApiWrapper and in google map types